README.md

```diff
@@ -125,33 +125,62 @@ yarn add -D jest-extended
 
 ## Setup
 
+```javascript
+// ./testSetup.js
+
+// add all jest-extended matchers
+import * as matchers from 'jest-extended';
```

Comment:
```suggestion
import matchers from 'jest-extended';
```

Looking at https://github.com/jest-community/jest-extended/blob/a0d7d65e997555f07c55b477c4b2ad13ae572b97/src/matchers/index.js#L5 it's a default export

Reply:
it _should_ be named though - thoughts on ditching `import-all.macro` and having explicit named exports?

Reply:
Ah. That's what I had originally but changed it because of an earlier comment, since I hadn't tested it.

I had wondered if that'd be better actually, though it would be a bit more verbose. Now that I think of it, the destructuring I added to the Readme might not work without named exports.

Reply:
Were you thinking of not using a default export in the index of each of matcher?

Reply:
I have no opinions on that, but `matchers/index.js` should explicitly import all files rather than using `import-all`